src/server/game/Movement/Spline/MoveSplineInit.h

```diff
@@ -53,6 +53,11 @@ namespace Movement
     public:
 
         explicit MoveSplineInit(Unit* m);
+        MoveSplineInit(MoveSplineInit&& init) : args(std::move(init.args))
```

Comment:
pretty sure you can just `MoveSplineInit(MoveSplineInit&& init) = default` here
also, you need to do the same for MoveSplineInitArgs (default one is not generated by compiler if you have "user-declared" destructors)

Reply:
daaaaaaaamn